index.html

```diff
@@ -1377,7 +1377,10 @@ <h4 id=el-caption>`caption`</h4>
     <tr>
       <th>Comments</th>
       <td>
-        If a descendant of a `table`, the first instance of a `caption` element will provide the `table` its accessible name.
+        <p class=note>
+          If a `caption` element is hidden from the accessibility tree, then it will not provide an accessible name
+          to its parent `table` element.
```

Comment:
```suggestion
          to its parent `table` element unless the `table` <a href="https://www.w3.org/TR/accname-1.2/#:~:text=user%20agents%20MUST%20include%20all%20nodes%20in%20the%20subtree%20as%20part%20of%20the%20accessible%20name%20or%20accessible%20description%2C%20when%20the%20node%20referenced%20by%20aria%2Dlabelledby%20or%20aria%2Ddescribedby%20is%20hidden">references it explicitly using `aria-labelledby`.</a>
```

If you can get past the ugliness of that deep link URL 🤦🏻‍♂️ , would it be too wordy or scope-creepy to call out [the `aria-labelledby` loophole](https://www.w3.org/TR/accname-1.2/#:~:text=user%20agents%20MUST%20include%20all%20nodes%20in%20the%20subtree%20as%20part%20of%20the%20accessible%20name%20or%20accessible%20description%2C%20when%20the%20node%20referenced%20by%20aria%2Dlabelledby%20or%20aria%2Ddescribedby%20is%20hidden) right here in each note?

Reply:
seems a very fragile way of linking. If we think we need to link to that we should create an idref to link to.

Reply:
Yeah, I’ve been a little deep-link-happy lately. 😅

[`https://www.w3.org/TR/accname-1.2/#comp_labelledby`](https://www.w3.org/TR/accname-1.2/#comp_labelledby) would get folks to the right neighborhood instead.

Reply:
i purposefully didn't add this into the notes, because it's not like using aria-labelledby makes the HTML element naming associations start working again.  Rather, the HTML naming associations never actually get trigged, since the aria-labelledby step has higher priority in the naming calculation.  

To put it another way, implementors aren't unclear about this aria-labelledby aspect now... this seems like it'd be more useful for authors to know.  So is this the best place to call this out?  I guess I'd be ok with a single note in the element naming section.  but, will authors come to this spec to understand this, or is this information better surfaced elsewhere?

Reply:
Yes, got it. 👍🏻  I think the accname spec already describes this behavior well enough for authors. The mapping clarifications in this PR combined with the “_If still empty_” conditions in html-aam’s 4.1 seem clear enough to me.

Reply:
glad that made sense.  thanks adam.  i'll leave this unresolved for a bit in case anyone else wants to weight in.  but otherwise i'll plan to mark this convo as resolved and not pull in this suggested change.